wordle-web/src/components/KeyBoard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <v-row class="justify-center" dense v-for="(key, i) in keyboardLetters" :key="i">
-    <v-col v-for="(letter, j) in key" :key="j">
+  <v-row rows="auto" class="justify-center" dense v-for="(key, i) in keyboardLetters" :key="i">
```

Comment:
I don't believe rows="auto" will change anything after testing on my code as well

Reply:
You're right, I just removed rows="auto" and it didn't change anything. Cols="auto" is the important part to get all the keys lined up together neatly.